src/lib/db/access-store.test.ts

```diff
@@ -104,3 +105,30 @@ test.each([
         expect(result).toStrictEqual(expected);
     },
 );
+
+describe('addAccessToProject', () => {
+    test.each(['roles', 'groups', 'users'])(
+        'should throw a bad data error if there is invalid data in the %s property of the addAccessToProject payload',
+        async (property) => {
+            const access = db.stores.accessStore as AccessStore;
+
+            const payload = {
+                roles: [4, 5],
+                groups: [],
+                users: [],
+                [property]: [123456789],
+            };
+
+            await expect(
```

Comment:
```
expect(() =>
    access.addAccessToProject(
        payload.roles,
        payload.groups,
        payload.users,
        'projectId',
        'createdBy',
    )
).rejects.toThrow(BadDataError);
```

Reply:
no need for await when you return a promise

Reply:
Mayyyybe. I'll try. I had some issues where I needed some of them, but maybe I added more than I need. I'll have a look.

Reply:
Yup, looks right. Done in 0408cd10c3eb6f5a9f9fa2ad367692ab0b60c7c2